compose.yaml

```
@@ -1,6 +1,6 @@
 services:
   postgres:
-    image: postgres:18
+    image: pgvector/pgvector:0.8.1-pg18-trixie
```

```suggestion
    image: pgvector/pgvector:pg18
```

See my comment for `ci.yml`.